src/Adapter/Admin/AbstractAdminQueryBuilder.php

```diff
@@ -166,6 +167,11 @@ protected function compileSqlQuery(array $select, array $table, array $where = a
             }
         }
 
+        // GROUP BY
+        if (!empty($groupBy)) {
+            $sql[] = 'GROUP BY '. implode(', ', $groupBy) . PHP_EOL;
```

Comment:
I feel we may probably handle user data here, and we need to add some sanitization here with

```php
array_map('pSQL', $groupBy);
```

Can you please replace this line with this one:
```php
$sql[] = 'GROUP BY '. implode(', ', array_map('pSQL', $groupBy)) . PHP_EOL;
```

<!-- ping @rGaillard -->